<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IRIScontrolClass</class>
 <widget class="QMainWindow" name="IRIScontrolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IRIScontrol</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget {
border: 3px solid gray;
border-radius: 40px;
background: white;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pBtnEnable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnDisable">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Target Position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eTargetPosition">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>qc</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBtnRelative">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Relative</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBtnAbsolute">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>160</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Absolute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtnMove">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>441</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual Value</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eTruePosition">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>qc</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eTargetPosition2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Node1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Node2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>qc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Node1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eTruePosition2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>qc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Node2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="iriscontrol.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>selectRelative()</slot>
 </slots>
</ui>
